internal/genny/build/build.go

```diff
@@ -50,7 +50,7 @@ func New(opts *Options) (*genny.Generator, error) {
 	g.Transformer(genny.Replace("/dot-", "/."))
 
 	// validate templates
-	g.RunFn(ValidateTemplates(os.DirFS(opts.App.Root), opts.TemplateValidators))
+	g.RunFn(ValidateTemplates(os.DirFS(opts.App.Root+"/templates"), opts.TemplateValidators))
```

Comment:
Does this need filepath.Join? (thinking on Windows separator char)

Reply:
Ah... I just thougth that could be handled inside golang. Not sure, actually I am very unfriendly with Windows... :-( Will fix that after quick check.

Reply:
I'm on the same boat. Actually having an issue with Windows on the other PR.

Reply:
It seems like `os.DirFS` internally handles it. The source file `src/os/file.go` also just uses "/" for joining.

https://github.com/golang/go/blob/a59ab29bf2164730e3e6d5439c6ec4b1e46adcc4/src/os/file.go#L658